L1Trigger/TrackFindingTracklet/interface/Settings.h

```diff
@@ -751,7 +753,8 @@ namespace trklet {
     unsigned int maxstepoffset_{0};
 
     //Number of processing steps for one event (108=18TM*240MHz/40MHz)
-    std::unordered_map<std::string, unsigned int> maxstep_{{"IR", 156},  //IR will run at a higher clock speed to handle
+    //IR should be set to 108 to match the FW for the summer chain, but ultimately should be at 156
+    std::unordered_map<std::string, unsigned int> maxstep_{{"IR", 108},  //IR will run at a higher clock speed to handle
```

Comment:
Settings.h should use the ultimate values, as the trigger group wants this. The doc https://github.com/cms-L1TK/firmware-hls/blob/master/README.md records which of these need changing when generating test data to match the current FW.

Reply:
done